tools/export/iar/iar_definitions.json

```diff
@@ -345,5 +345,8 @@
     "M2351KIAAEES": {
         "OGChipSelectEditMenu": "M2351 series\tNuvoton M2351 series",
         "TrustZone": 1
+    },
+    "S5JS100": {
+        "OGChipSelectEditMenu": "S5JS100\tSamsung S5JS100"
     }
-}
+} 
```

Comment:
Looks like a space has slipped in (sorry for nitpicking).

Reply:
Addressed from 4a063328bb2f02333932669e00fa7d7a512e215e